sherpa/astro/xspec/tests/test_xspec_unit.py

```diff
@@ -402,6 +402,86 @@ def test_abund_element_z():
     assert fe == pytest.approx(2.69e-05)
 
 
+@requires_xspec
+def test_abund_element_named():
+    """Can we access the elemental settings?
+    """
+
+    from sherpa.astro import xspec
+
+    # These values were found from HEASOFT version 6.19
+    # spectral/manager/abundances.dat
+    # The values are given to two decimal places in this file.
+    # It is not worth testing all settings, since we are not
+    # testing the XSPEC implementation itself, just our use of it.
+    #
+    h = xspec.get_xsabund('H', 'wilm')
+    he = xspec.get_xsabund('He', 'wilm')
+    si = xspec.get_xsabund('Si', 'wilm')
+    ar = xspec.get_xsabund('Ar', 'wilm')
+    k = xspec.get_xsabund('K', 'wilm')
+    fe = xspec.get_xsabund('Fe', 'wilm')
+
+    assert h == pytest.approx(1.0)
```

Comment:
The test logic should be moved into a subroutine as it's used three times and we want to avoid repetition and make it more obvious where the same thing is being checked